crates/deno-subsystem/deno-model-builder/src/system_builder.rs

```diff
@@ -93,43 +93,70 @@ fn process_script(
 ) -> Result<(String, Vec<u8>), ModelBuildingError> {
     module_skeleton_generator::generate_module_skeleton(module, base_system, module_fs_path)?;
 
+    fn run_local<F, R>(future: F) -> R
+    where
+        F: std::future::Future<Output = R>,
+    {
+        let rt = tokio::runtime::Builder::new_current_thread()
+            .enable_io()
```

Comment:
Maybe we should use `enable_all()` here (based on the failure seen in Windows CI) https://github.com/exograph/exograph/actions/runs/5731230841/job/15534564500#step:10:213

Reply:
I made the change and that helped build on Windows and mac-86. Linux is still failing, though.